modules/installation/pages/setup-wizard.adoc

```diff
@@ -72,5 +73,6 @@ Synchronization progress for each product is shown in a progress bar next to the
 Depending on the products you have chosen, synchronization can take up to several hours.
 New products will be available for you to use in {productname} when synchronization is complete.
 
-If your synchronization fails, it could be because of a third party GPG key.
+If your synchronization fails, it could be because of a third party GPG key or your company firewall block access to the download server.
```

Comment:
```suggestion
If your synchronization fails, it could be because of a third party GPG key or your company firewall blocking access to the download server.
```